aviatrix/resource_aviatrix_aws_tgw.go

```diff
@@ -181,9 +181,19 @@ func resourceAviatrixAWSTgw() *schema.Resource {
 					"Valid values: true, false. Default value: true.",
 			},
 			"enable_multicast": {
-				Type:     schema.TypeBool,
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Enable Multicast.",
+			},
+			"cidrs": {
```

Comment:
1. May need check if backend preserve the order of the input for "cidrs";
2. May also check if "cidrs" will be referenced somewhere. If yes, the order of the cidrs may matter.

Reply:
1. Order that is input into the UI is not preserved.
2. In the TGW Connect workflow, the CIDRs are not referenced.